fr/part2.md

```diff
@@ -2,9 +2,9 @@
 
 Le protocole HTTP 1.1 est omniprésent sur Internet. De nombreux investissements ont été réalisés sur des protocoles et infrastructures tirant profit de celui-ci. À tel point que lors de l'implémentation d'un nouveau projet, il est souvent plus facile d'utiliser HTTP plutôt que de développer un nouveau protocole.
 
-## 2.1 HTTP 1.1 est énorme
+## 2.1 HTTP 1.1 est dense
```

Comment:
Like @machour I don't like the translation using the french "dense" for the english "huge". I think the previous "énorme" was better. wordreference also tells me "immense" which I like too, so please pick either `énorme` or `immense`, I believe this is closer to the intended meaning.

Reply:
I got it but, in my opinion, "énorme" and "immense" are more likely to be used to describe physical hugeness instead of an abstract one (the hugeness of the HTTP 2 specification).

Also, when I read "énorme" I just automatically think about people saying "C'est énorme !" in the sense that "This thing is so cool!". That's also why I wanted to replace this "énorme" with something else. 

I like the idea of @fantuz: adding "très dense".
I think it will suffice to convey the original meaning of "huge" in this case.

Do you agree?